lib/puppet_x/certs/common.rb

```diff
@@ -42,6 +42,8 @@ module Common
 
         newparam(:password_file)
 
+        newparam(:build_dir)
```

Comment:
Should this have a default value? We should also verify that it's not empty, right? I don't think the tool underneath it would handle that well.

Reply:
I added both just to make it easier. The Jenkins trick does look a bit magical and I'd rather just be explicit.